docs/bugs.json

```
@@ -1,4 +1,13 @@
 [
+    {
+        "uid": "SOL-2022-6",
+        "name": "CorruptABIEncodingWithYul",
```

How about this:
```suggestion
        "name": "ABIEncoderV2HeadOverflowWithStaticArrayCleanup",
```
It's kinda like a buffer overflow :)

 Also it's reproducible without Yul, isn't it?